tests/unit/test_exceptions.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from histolab import exceptions as exp
+
+from ..unitutil import ANY, initializer_mock
+
+
+class DescribeExceptions(object):
+    @pytest.mark.parametrize("args", ("error", ["hello", "error"], None))
+    def it_can_construct_from_args_level_error(self, request, args):
+        _init = initializer_mock(request, exp.LevelError)
+
+        if type(args) == list:
```

Comment:
In general is not a best practice using `if(s)` in test functions. So i suggest to refactor the test of write different test according to all the cases you have